app/mailers/devise_mailer.rb

```diff
@@ -2,7 +2,7 @@ class DeviseMailer < Devise::Mailer
   helper :application # gives access to all helpers defined within `application_helper`.
   include Devise::Controllers::UrlHelpers # Optional. eg. `confirmation_url`
 
-  def reset_password_instructions(record, token, opts={})
+  def reset_password_instructions(record, token, _opts = {})
```

Comment:
If `opts` is not used at all, either remove it entirely or, if the expected signature requires it explicitly, don't assign a default.

Reply:
the signature in the parent class is `reset_password_instructions(record, token, opts={})`. Just the same signature